plugins/inputs/t128_graphql/t128_graphql_test.go

```diff
@@ -307,6 +309,54 @@ var ResponseProcessingTestCases = []struct {
 		},
 		ExpectedErrors: nil,
 	},
+	{
+		Name:       "processes response with nested tags",
```

Comment:
The response I'm assuming is somehow injected via a mock, correct? Can you add tests where we can verify the query being generated for these inputs?